File: src/slash-commands/ping.js
import { SlashCommandBuilder } from 'discord.js';

export default {
  data: new SlashCommandBuilder()
    .setName('ping')
    .setDescription('Répond pong (slash) 🏓'),
  async execute(interaction) {
    await interaction.reply('Pong 🏓 (slash)');
  }
};

File: Python/slash_commands/ping.py
"""Slash /ping"""
name = 'ping'
description = 'Répond pong (slash) 🏓'

async def execute(interaction, **kwargs):
    try:
        await interaction.response.send_message('Pong 🏓 (slash)')
    except Exception:
        try:
            if not interaction.response.is_done():
                await interaction.followup.send('Pong 🏓 (slash)')
        except Exception:
            pass
